Testing/functional/e2e/cypress/integration/timeline/filter.js

```diff
@@ -1,13 +1,14 @@
 const { AuthMethod, localDevUri } = require("../../support/constants");
 
 function verifyActiveFilter(activeFilterCount) {
-    cy.get('[data-testid=filterDropdown]').click();
+    cy.get("[data-testid=filterDropdown]").click();
     cy.get('[data-testid=filterDropdown]').should('have.css', 'background-color', 'rgb(0, 146, 241)'); // has the '#0092f1' background-color
     cy.get('[data-testid=filterDropdown] > span').contains(activeFilterCount); // has 1 active filter
     cy.viewport('iphone-6');
     cy.get('[data-testid=mobileFilterDropdown]').should('have.css', 'background-color', 'rgb(0, 146, 241)'); // has the '#0092f1' background-color
     cy.viewport(1000, 600);
-    cy.get('[data-testid=filterDropdown]').click();
+    cy.wait(500);
```

Comment:
Many people are complaining about the click not working in cypress. Waiting a bit before clicking (if it fails) is a work-around.

Reply:
This seems like an anti-pattern of cypress, we shouldn't have to wait to click.

Reply:
yeah, it seems it is an issue with cypress, reading a lot of their threads on their github, they couldn't suggest any solid ways for this scenario.